frontend/explorer/public/js/main.js

```diff
@@ -291,4 +307,4 @@ function buildPageTitle() {
 			break;
 	}
 }
-buildPageTitle();
+buildPageTitle();
```

Comment:
Did you remove a newline here? revert change please.

Reply:
Re-added newline